<launch>
  <arg name="frame_id" default="TanwayTP" />

  <node pkg="tensorpro_view" name="tensorpro_node" type="tensorpro_node" output = "screen">
    <param name="frame_id" value="$(arg frame_id)" />

    <param name="port"  type="int" value="5600" />
    <param name="host"  type="string" value="192.168.111.204" />
    <param name="LiDARport"  type="int" value="5050" />
    <param name="LiDARhost"  type="string" value="192.168.111.51" />
    <param name="StartAngle"  type="double" value="30" />
    <param name="EndAngle"  type="double" value="150" />

    <param name="min_range"  type="double" value="0" />
    <param name="max_range"  type="double" value="200" />
    <param name="min_channel"  type="int" value="1" />
    <param name="max_channel"  type="int" value="16" />

   </node>

   <node name="rviz" pkg="rviz" type="rviz" args="-d $(find tensorpro_view)/rviz/tensorpro.rviz" />
</launch>
